src/gatsby-node.js

```diff
@@ -97,6 +130,12 @@ const processData = (data, baseUrl) => {
     }
   });
 
+  if (node.text) {
+    if (node.text['content-type'] === 'text/html') {
+      node.text.html = processHtml(node.text.data, baseUrl);
```

Comment:
maybe node.text.react =

Reply:
There's also node.text.jsx? What would be better here?